app/views/admin/overview/index.html.erb

```diff
@@ -22,6 +22,17 @@
         </div>
       </div>
     </section>
+    <h2 class="crayons-subtitle-1 mb-3">Analytics</h2>
+    <article class="grid s:grid-cols-2 gap-3 mb-4" data-controller="ahoy">
+      <section class="crayons-card">
+        <div class="crayons-card__body">
+          <h2 class="crayons-title"><%= @open_abuse_reports_count %></h2>
+          <p class="crayons-subtitle-4 color-base-70">
+            Open reports
```

Comment:
just a tiny nitpick, but do want to add some code here so that we show:

`1 Open report`

instead of `1 Open reports`?

Reply:
I think you can pass the count argument to String#pluralize?

```
<%= "Open report".pluralize(@open_abuse_reports_count) %>
```

That appears to do the right thing (when count == 1 it does _not_ pluralize):

```
[4] pry(main)> "Open report".pluralize(1)
=> "Open report"                         
[5] pry(main)> "Open report".pluralize(2)
=> "Open reports"                        
[6] pry(main)> "Open report".pluralize(0)
=> "Open reports"
```